native_client/client.cc

```diff
@@ -87,10 +133,17 @@ main(int argc, char **argv)
   sox_close(input);
 
   // Pass audio to DeepSpeech
-  char* result = DsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
+  struct ds_result* result = LocalDsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
   if (result) {
-    printf("%s\n", result);
+    printf("%s\n", result->string);
```

Comment:
Whoops, I actually missed something here - it's possible for result->string to be NULL, so there's now a possible crash - the free(result->string) and print of the string should be in a block that checks it isn't NULL first.